### PR TITLE
devops: add test coverage

### PR DESCRIPTION
**Pull request by awentzel**
<body>
details below
<footer>
https://docs.codeclimate.com/docs/configuring-test-coverage

### CODE REVIEW

**Comment by awentzel** [comment]
Testing still.  Do not code review yet.

File: packages/fast-jss-manager-angular/package.json [hunk 1]
```diff
@@ -6,33 +6,36 @@
...
+    "unit-tests": "jest",
+    "unit-tests:watch": "jest --watch"
   },
-  "license": "MIT",
-  "readme": "README.md",
   "jest": {
+    "collectCoverage": true,
+    "moduleFileExtensions": [
+      "ts",
+      "js",
```
Comment (nicholasrice) [inline packages/fast-jss-manager-angular/package.json]: just curious - why does this need to resolve JSON?

File: packages/fast-jss-manager-react/package.json [hunk 2]
```diff
@@ -21,23 +23,31 @@
     "unit-test": "jest",
     "unit-test:watch": "jest --watch && npm-watch"
   },
-  "license": "MIT",
-  "readme": "README.md",
   "jest": {
+    "collectCoverage": true,
+    "coverageThreshold": {
```
Comment (nicholasrice) [inline packages/fast-jss-manager-react/package.json]: why do only some of these have coverage thresholds and why are the ones that do differ from each-other?

File: packages/fast-viewer-react/package.json [hunk 3]
```diff
@@ -12,20 +26,36 @@
...
+      "src"
+    ],
+    "globals": {
+      "ts-jest": {
+        "tsConfigFile": "./src/tsconfig.json"
+      }
+    }
   },
-  "license": "MIT",
+
```
Comment (nicholasrice) [inline packages/fast-viewer-react/package.json]: nit: extra line

File: packages/fast-jss-manager-react/package.json (hunk 2 above)
Comment (awentzel) [reply packages/fast-jss-manager-react/package.json]: I added coverage thresholds to ensure we do not slip and get worse. But, ultimately, we need to discuss as a team our code coverage metrics and/or test coverage goals and meet those.

File: packages/fast-jss-manager-angular/package.json (hunk 1 above)
Comment (awentzel) [reply packages/fast-jss-manager-angular/package.json]: I added JSON as a we use these type of files https://facebook.github.io/jest/docs/en/configuration.html#modulefileextensions-array-string

File: packages/fast-viewer-react/package.json (hunk 3 above)
Comment (awentzel) [reply packages/fast-viewer-react/package.json]: fixed

**Pull request merged**